### PR TITLE
feat(Operator) Remove translations

### PR DESCRIPTION
**Pull request by santostiago**
Remove the translations and moves it to the Operator.

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [x] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: https://www.pivotaltracker.com/story/show/183775139

### CODE REVIEW

**Review by tsubik — CHANGES_REQUESTED** [review]
@santostiago I don't see migration code here

File: app/models/operator.rb [hunk 1]
```diff
@@ -109,16 +100,6 @@ class Operator < ApplicationRecord
     end
   }
 
-  ransacker(:name) { Arel.sql('operator_translations.name') } # for nested_select in observation form
-
-  class Translation
-    after_save do
-      if saved_change_to_name? && locale == :en
```
[the remark this answers is not in the document]
Comment (santostiago) [reply app/models/operator.rb:116]: > @santostiago I don't see migration code here

I just noted it was not added to the commit 😨 
Adding it now.

[the remark this answers is not in the document]
Comment (santostiago) [reply app/models/operator.rb:116]: >this hook was to update fmu geojson properties with new operator's name, so you have to re-add this hook to operator model.
Done.

**Review by tsubik — APPROVED** [review]
Looks great :+1:

**Pull request merged**